src/types.ts

```diff
@@ -143,35 +144,6 @@ export interface UpdateOptions {
   // session?: ClientSession
 }
 
-/**
- * interface for WriteConcern documents used by MongoDB
- *
- * @see https://docs.mongodb.com/manual/reference/write-concern/
- */
-export interface WriteConcern {
```

Comment:
Why `WriteConcern` interface deleted?

Reply:
It's not I just moved it out of the massive types declare file to readwriteconcers that's how the spec is also designed